R/nullModelAsMatrix.R

```diff
@@ -0,0 +1,27 @@
+.checkMatrixType <- function(covMatList) {
+    isMatrix <- sapply(covMatList, is, "Matrix")
+    if (any(isMatrix) & !all(isMatrix)) {
+        for (i in which(isMatrix)) {
```

Comment:
shouldn't this be for i in which they are not isMatrix (i'm having a hard time thinking how to write that in code at the moment)

Reply:
Yes, it should.